Source/Core/getBaseUri.js

```diff
@@ -0,0 +1,38 @@
+/*global define*/
+define([
+        './defined',
+        './DeveloperError'
+    ], function(
+        defined,
+        DeveloperError) {
+    "use strict";
+
+    /**
+     * Given a URI, returns the base path of the URI.
+     * @exports getBaseUri
+     *
+     * @param {String} uri The Uri.
+     * @returns {String} The base path of the Uri.
+     *
+     * @example
+     * // basePath will be "/Gallery/";
+     * var basePath = Cesium.getBaseUri('/Gallery/simple.czml?value=true&example=false');
+     */
+    var getBaseUri = function(uri) {
```

Comment:
Add this to CHANGES.md or I would be fine with it being `@private` (and also leave the existing doc).